src/test/java/de/retest/recheck/cli/subcommands/CommitIT.java

```diff
@@ -33,12 +33,12 @@
 
 	@Test
 	public void commit_without_argument_should_return_the_usage_message() {
-		final String expectedMessage = "Usage: commit [--all] [--exclude=<exclude>]... <testReport>\n"
-				+ "Accept specified differences of given test report.\n"
+		final String expectedMessage = "Usage: commit [--all] [--exclude=<exclude>]... <testReport>\n" + "\n"
```

Comment:
Regarding a new line: Should every `\n` be on an actual new line (maybe except `\n\n`)?

Reply:
I changed it so it will look as close as possible as the actual output, is that in line with what you envisoned?